scripts/docker-helper-4.x/cht-docker-compose.sh

```diff
@@ -176,6 +183,15 @@ fi
 if [[ -n "${2-}" && -n $projectName ]]; then
 	containerIds=$(docker ps --all --filter "name=${projectName}" --quiet)
 	case $2 in
+ 	"init")
```

Comment:
`init` doesn't seem to work :

https://github.com/medic/cht-core/assets/8253488/fa4392e9-87cc-4d03-b565-c9bd681777d8